cmd/clusterawsadm/cmd/alpha/bootstrap/bootstrap.go

```diff
@@ -207,15 +207,15 @@ func generateIAMPolicyDocJSON() *cobra.Command {
 				SharedConfigState: session.SharedConfigEnable,
 			})
 			if err != nil {
-				return fmt.Errorf("Error: failed to create a session: %v", err)
+				return fmt.Errorf("ERROR: failed to create a session: %v", err)
```

Comment:
Remove the `ERROR:` prefix. The convention in cluster-api is to return errors that are lower-cased and not prefixed.

Reply:
It's 5 occurrences of `fmt.Errorf("error: ....")` in this project, I'll fix all 5 of them in the same commit.